crates/sui-core/src/authority/authority_store.rs

```diff
@@ -269,7 +269,10 @@ impl<S: Eq + Serialize + for<'de> Deserialize<'de>> SuiDataStore<S> {
     }
 
     /// Acquire the lock for a tx without writing to the WAL.
-    pub async fn acquire_tx_lock<'a>(&'a self, digest: &TransactionDigest) -> CertLockGuard<'a> {
+    pub async fn acquire_tx_lock<'a, 'b>(
+        &'a self,
+        digest: &'b TransactionDigest,
```

Comment:
How about we pass it by value not ref?

Reply:
These extraneous lifetimes should go away now that we are on rust-1.62.1 - there was a bug with incremental compilation and async lifetimes that we were trying to work around.